### DISCUSSION TITLE
Control bar toggle

### DISCUSSION THREAD

**Discussion by clfberlin** [seaky/nspanel_pro_tools_apk#58] [General]
Really nice! I'm looking forward to the direct HA connection. Is there a way to toggle the control bar? That is the main reason I am using Fully Kiosk at the moment.
Thanks and cheers

**Comment by seaky** [comment]
Hi,
What do you mean by control-bar? Navigation bar? Notification bar?

**Reply by clfberlin** [reply]
Sorry. Yes, navigation plus volume control at the bottom of the screen.

**Comment by seaky** [comment]
I did some resarch and relatively easy implement, so I'll plan it to somewhere.
Can you make an issue thus I can assign it for a release. thx

**Reply by clfberlin** [reply]
Glad to do so. Thanks!

**Comment by Doug74000** [comment]
Hi, Did you implement this function?
I saw in one of your video that you you can do it with your panel but impossible to find how...
Many thanks for your work!

**Reply by Doug74000** [reply]
Sorry, just saw that it will be implemented in v2.2...
